apps/client/src/shared/component/Header/Notification/NotificationTab.tsx

```diff
@@ -0,0 +1,66 @@
+import { useNotificationsQueryObject } from "@/app/api/notifications/query";
+import type { NotificationType } from "@/shared/component/Header/Notification";
+import {
+  indicatorStyle,
+  notificationTabStyle,
+  textStyle,
+} from "@/shared/component/Header/Notification/Notification.css";
+import { useQueryClient } from "@tanstack/react-query";
+import { motion } from "framer-motion";
+import type { Dispatch, SetStateAction } from "react";
+
+type NotificationTabProps = {
+  tabId: NotificationType;
+  notificationType: NotificationType;
+  setNotificationType: Dispatch<SetStateAction<NotificationType>>;
+  shrinkList: () => void;
+
+  children: string;
+};
+
+const NotificationTab = ({
+  tabId,
+  notificationType,
+  setNotificationType,
+  shrinkList,
+
+  children,
+}: NotificationTabProps) => {
+  const queryClient = useQueryClient();
+
+  const prefetchNotification = (notificationType: NotificationType) => {
+    // biome-ignore lint/correctness/useHookAtTopLevel: <explanation>
+    queryClient.prefetchQuery(useNotificationsQueryObject(notificationType));
+  };
+
+  const isSelected = notificationType === tabId;
+
+  const handleTabClick = () => {
+    setNotificationType(tabId);
+    shrinkList();
+  };
+
+  return (
+    <li
+      onMouseEnter={() => prefetchNotification(tabId)}
+      onFocus={() => prefetchNotification(tabId)}
+    >
+      <button
+        role="tab"
+        aria-selected={isSelected}
+        className={notificationTabStyle({ isSelected })}
+        onClick={handleTabClick}
+      >
```

Comment:
```suggestion
    <li>
      <button
        role="tab"
        aria-selected={isSelected}
        className={notificationTabStyle({ isSelected })}
        onClick={handleTabClick}
        onMouseEnter={prefetchNotification}
        onFocus={prefetchNotification}
      >
```
`<li>` 요소는 button과 달리 `tabindex`의 적용을 받지 않으니까 focus 되지 않아요. 그러니 이벤트 핸들러들을 button으로 옮겨야 하고(MouseEnter는 작동하지만 같이 옮겨주죠), `prefetchNotification` 함수도 인자를 받기 보다는 내부의 QueryObject에 tabid를 바로 주는 식으로 변경하는 게 좋겠어요.